Nodejs/Product/Nodejs/Debugger/DebugEngine/AD7Engine.cs

```diff
@@ -482,8 +483,8 @@ public sealed class AD7Engine : IDebugEngine2, IDebugEngineLaunch2, IDebugProgra
         // in which case Visual Studio uses the IDebugEngineLaunch2::LaunchSuspended method
         // The IDebugEngineLaunch2::ResumeProcess method is called to start the process after the process has been successfully launched in a suspended state.
         int IDebugEngineLaunch2.LaunchSuspended(string pszServer, IDebugPort2 port, string exe, string args, string dir, string env, string options, enum_LAUNCH_FLAGS launchFlags, uint hStdInput, uint hStdOutput, uint hStdError, IDebugEventCallback2 ad7Callback, out IDebugProcess2 process) {
-            DebugWriteLine("--------------------------------------------------------------------------------");
-            DebugWriteLine("AD7Engine LaunchSuspended Called with flags '{0}' ({1})", launchFlags, GetHashCode());
+            LiveLogger.WriteLine("--------------------------------------------------------------------------------");

```

Comment:
Why the row of dashes above this particular emit (and none of the others)? Seems unnecessary.